nipype/algorithms/confounds.py

```diff
@@ -307,9 +314,20 @@ def _run_interface(self, runtime):
             axis=1,
             arr=mpars,
             source=self.inputs.parameter_source)
-        diff = mpars[:-1, :6] - mpars[1:, :6]
-        diff[:, 3:6] *= self.inputs.radius
-        fd_res = np.abs(diff).sum(axis=1)
```

Comment:
This L1 norm is what's used in the cited paper, so I think we should not replace it.

What if we instead add a `metric` option to the input spec, with `L1` and `geodesic` (or is there a better name for this metric) as options?